scripts/variantstore/scripts/run_avro_query_for_sample_info.py

```diff
@@ -32,13 +32,14 @@ def get_number_of_partitions(dataset_name, project_id):
     return math.ceil(max_table_num[0])
 
 
-def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix):
+def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix, new_sample_cutoff=None):
     num_of_tables = get_number_of_partitions(dataset_name, project_id)
 
     for i in range(1, num_of_tables + 1):
         file_name = f"*.{i:03}.avro"
         id_where_clause = f"sample_id >= {((i -1) * 4000) + 1} AND sample_id <= {i * 4000}"
-
+        if new_sample_cutoff:
+            id_where_clause += f"sample_id >= {((i -1) * 4000) + 1} AND sample_id <= {i * 4000} AND sample_id < {new_sample_cutoff}"
```

Comment:
don't we want this to be sample_id > new_sample_cufoff or am I reading this incorrectly? I think we want all new samples greater than a certain id, because those are the new one.

Also, as an aside, can we limit the number of tables that we scan over based on the sample_id / 4000 being the first table in which we'll actually see samples?  That _should_ be reliable to do here, right?  Basically, just short circuiting this loop if i < sample_id / 4000 to avoiding creating empty avro files for a bunch of tables that we know aren't going to have any sample data that we want.